### PR TITLE
EventGrid: CLI support for 2019-06-01 API Version

### PR DESCRIPTION
**Pull request by ahamad-MS**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-eventgrid/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 Release History
 ===============
+0.2.4
++++++
+* `domain create/update/delete/get/list`: Added new commands for the domain related CRUD operations.
+* `domain topic create/update/delete/get/list`: Added new commands for the domain topics related CRUD operations.
+* `topic list": Added --odata-query for filtering the list results using OData syntax.
```
Comment (limingu) [inline src/command_modules/azure-cli-eventgrid/HISTORY.rst]: Need to replace " with `

File: src/command_modules/azure-cli-eventgrid/HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,15 @@
...
+* `event-subscription list": Added --odata-query for filtering the list results using OData syntax.
```
Comment (limingu) [inline src/command_modules/azure-cli-eventgrid/HISTORY.rst]: Need to replace " with `

**Comment by ahamad-MS** [comment]
@yugangw-msft  @tjprescott  Can you please help assigning someone to this PR and help us understanding the failure in the checks here. It is not clear to me what is failing and some of them is not actionable (for example, checking to master is not allowed. Thanks.

**Comment by yugangw-msft** [comment]
The failure was caused by a test flakiness which we fixed yesterday. Please rebase

**Comment by ahamad-MS** [comment]
@yugangw-msft  I did the following steps and I still get the same failure.

git fetch upstream
git merge upstream/master

There was nothing to commit but I made simple editorial change in history to kick off new build and I am still hitting the same unrelated failures.

Any idea?

**Comment by ahamad-MS** [comment]
CLosing this one and restarting new PR for dev branch

**Pull request closed without merging**